bin/node/src/commands.rs

```diff
@@ -94,21 +118,35 @@ impl BootstrapChainCmd {
 
         Ok(())
     }
+}
 
-    fn open_keystore(&self, authority: &str, chain_id: &str) -> Result<Arc<LocalKeystore>, Error> {
-        let base_path: BasePath = self.chain_params.base_path().path().join(authority).into();
+/// The `bootstrap-node` command is used to generate key pairs for a single authority
+/// private keys are stored in a specified keystore, and the public keys are written to stdout.
+#[derive(Debug, StructOpt)]
+pub struct BootstrapNodeCmd {
+    #[structopt(long)]
+    pub authority: String,
 
-        info!(
-            "Writing to keystore for authority {} and chain id {} under path {:?}",
-            authority, chain_id, base_path
+    #[structopt(flatten)]
+    pub keystore_params: KeystoreParams,
+
+    #[structopt(flatten)]
+    pub chain_params: ChainParams,
+}
+
+impl BootstrapNodeCmd {
+    pub fn run(&self) -> Result<(), Error> {
+        let authority = &self.authority;
+        let keystore = open_keystore(
+            &self.keystore_params,
+            &self.chain_params,
+            authority.as_str(),
```

Comment:
use `get_account_id_from_seed` and pass authority or node-name arguments as seed. This will result in behaviour consistent with the BootstrapChainCommand, where the keystore path is equal to the accountId